mods/ts/rules/defaults.yaml

```diff
@@ -566,6 +569,9 @@
 	Carryable:
 		RequiresCondition: !inside-tunnel
 	RevealOnFire:
+	RejectsOrders@tunnel:
+		Reject: Stop
```

Comment:
That is probably the most efficient way forward.  The "proper" fix that I have in mind will depend on the keepQueue flag from #12919, so it would make more sense to get this and that merged and then revisit the idea again in its own PR.